lib/plugins/EPrints/Plugin/Import/PubRouter.pm

```diff
@@ -322,26 +311,17 @@ sub xml_to_epdata
 	}
 
 	#now loop around again, now that DOI has been handled
-	my @id_notes;
 	foreach my $id (@identifiers)
 	{
 		my $identifier_type = $id->getAttribute( "type" );
 		if( $identifier_type ne "doi" )
 		{
-			if( defined $epdata->{id_number} )
-			{
-				push @id_notes, "$identifier_type: " . $plugin->xml_to_text( $id );
-			} 
-			else
+			if( !defined $epdata->{id_number} )
```

Comment:
I think the code from lines 313-324 can be removed now, if 304-311 is changed:
```
	foreach my $id (@identifiers)
	{
		my $identifier_type = $id->getAttribute( "type" );
                $epdata->{id_number} = $plugin->xml_to_text( $id );
		last if( $identifier_type eq "doi" );
	}
```

Reply:
Quite right - this is a much nicer way of doing - see https://github.com/jisc-services/eprints-plugin-PubRouter-RIOXXplus-Connector/pull/5/commits/5d2140888ed57771e75f7fe23225da064962ed62